src/Text/Pandoc/Writers/Markdown/Inline.hs

```diff
@@ -116,9 +122,56 @@ attrsToMarkdown attribs = braces $ hsep [attribId, attribClasses, attribKeys]
               escAttrChar '\\' = literal "\\\\"
               escAttrChar c    = literal $ T.singleton c
 
+attrsToMarkua:: Attr -> Doc Text
+attrsToMarkua attributes
+     | null list = empty
+     | otherwise = braces $ intercalateDocText list
+        where attrId = case attributes of
+                        ("",_,_) -> []
+                        (i,_,_)  -> [literal $ "id: " <> i]
+              -- all non explicit (key,value) attributes besides id are getting
+              -- a default class key to be Markua conform
+              attrClasses = case attributes of
+                             (_,[],_) -> []
+                             (_,classes,_) -> map (escAttr . ("class: " <>))
+                                classes
+              attrKeyValues = case attributes of
+                               (_,_,[]) -> []
+                               (_,_,keyvalues) -> map ((\(k,v) -> escAttr k
+                                              <> ": " <> escAttr v) . 
+                                              preprocessKeyValues) keyvalues
+              escAttr          = mconcat . map escAttrChar . T.unpack
+              escAttrChar '"'  = literal "\""
+              escAttrChar c    = literal $ T.singleton c
+
+              list = concat [attrId, attrClasses, attrKeyValues]
+
+              -- if attribute key is alt, caption, title then content
+              -- gets wrapped inside quotes
+              -- attribute gets removed
+              preprocessKeyValues :: (Text, Text) -> (Text, Text)
+              preprocessKeyValues (key,value)
+                 | key == "alt" ||
+                   key == "caption" ||
+                   key == "title" = (key, inquotes value)
+                 | otherwise = (key,value)
+              intercalateDocText :: [Doc Text] -> Doc Text
+              intercalateDocText [] = empty
+              intercalateDocText [x] = x
+              intercalateDocText (x:xs) = x <> ", " <> (intercalateDocText xs)
+
+-- | Add a (key, value) pair to Pandoc attr type
+addKeyValueToAttr :: Attr -> (Text,Text) -> Attr
+addKeyValueToAttr (attrlistId,classes,keyvalues) (key,value) 
+    | not (T.null key) && not (T.null value) = (attrlistId,
+                                                classes,
+                                                (key,value): keyvalues)
+    | otherwise = (attrlistId,classes,keyvalues)
```

Comment:
For consistency, maybe use `ident` instead of `attrlistId`?

Reply:
we changed it to fit to the others